Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Data/Security/1030-TPDM-DeletePerformanceEvaluationRatingsAuthorizationStrategy.sql

```diff
@@ -0,0 +1,23 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE 
```

Comment:
First get parent resource claim id  for performanceEvaluationRating resource claim and check parent resource claim PerformanceEvaluation  exist with record and then delete performanceEvaluationRatings if exists

Reply:
Apply this logic for postgres also